Lung Segmentation/models.py

```diff
@@ -75,7 +76,7 @@ def BCDU_net_D3(input_size = (256,256,1)):
     conv8 = Conv2D(2, 3, activation = 'relu', padding = 'same', kernel_initializer = 'he_normal')(conv8)
     conv9 = Conv2D(1, 1, activation = 'sigmoid')(conv8)
 
-    model = Model(input = inputs, output = conv9)
+    model = Model(inputs, conv9)
```

Comment:
There was an error with `Tensorflow 2.x` which is fixed by using positional parameters.